src/tests/src/libs/antares/array/tests-matrix-load.cpp

```diff
@@ -0,0 +1,685 @@
+#define BOOST_TEST_MODULE test-lib-antares-matrix tests
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <boost/test/included/unit_test.hpp>
+
+#include<array/tests-matrix-load.h>
+
+#include <iostream>
+#include <stdio.h>
+
+namespace utf = boost::unit_test;
+
+using namespace Yuni;
+
+/*
+All loadFromCSVFile(...) entries (some directions to test this big method):
+	0. Global JIT::enabled state : on or off
+	1. file to be loaded (= buffer) :
+		a. empty
+		b. contains the head banner 'size:nxm' or not
+		c. contains digits (which precision) or integers.
+		d. full of zeros,
+		e. full of numbers (> 0 or < 0)
+		f. strange cases
+	2. Initial matrix state (matrix state when loadFromCSVFile(...) is called) :
+		a. empty
+		b. sized n x m
+		c. state of the jit
+	3. loadFromBuffer(...) method arguments :
+		a. desired final matrix size (minWidth & maxHeight)
+		b. options :
+			+ optNone = 0,				//! None
+			+ optFixedSize = 1,			//! The matrix can not see its size modified
+			+ optQuiet = 2,				//! Do not produce warnings/errors
+			+ optImmediate = 4,			//! Do not postpone the loading
+			+ optMarkAsModified = 8,	//! mark the matrix as modified after loading
+			+ optNoWarnIfEmpty = 16,	//! Do not warn if the file is empty
+			+ optNeverFails = 32,		//! The loading never fails
+	4. Error type returned by loadFromFileToBuffer(...)
```

Comment:
La classe possède beaucoup de test et il est utile d'avoir un résumé en première partie.
Il serait aussi intéressant d'avoir ces informations juste avant le test.

Reply:
Normalement, le titre du test renseigne sur ce qu'il fait, mais ce n'est peut-être pas suffisant...

Reply:
OK je n'avais pas noté.